Add English texts to CONTRIBUTING.md. Modify Japanese texts. (#14)
rt-net/.github: CONTRIBUTING.md

> # Contribution Guide

Reviewer: ```suggestion
# Contribution Guide (EN)
```

This guide describes how to contribute to the [rt-net](https://github.com/rt-net) organization repository.

## Issues

Thank you for helping us improve the quality of the repository.

We have prepared a template to make it easier for you to create issues.

## Pull Requests

Thank you for creating Pull Requests.

The following rules apply to your Pull Request (PR)

- The content of the PR is subject to the license of each repository (as described in the `LICENSE` or `README.md`).
- PRs are reviewed by `rt-net` members before being merged

>   - Not all PRs will be merged, and some may not meet your expectations.

Reviewer 2: ```suggestion
  - Not all PRs will be merged and the review process may take some time.
```

> - If you have a test set up in the repository, please test it as much as possible.

Reviewer 2: ```suggestion
- For requests to repositories where CI and tests are set up, please try to pass the tests as much as possible.
```

>   - If you submit a PR without testing for some reason (e.g., if there is a mistake in the testing), please indicate this in the PR.

Reviewer: ```suggestion
  - If you submit a PR without testing for a specific reason (e.g., if there is a mistake in the testing), please describe it in the PR.
```

- When merging, all commits in the PR will be `squashed` into one commit.

> - Keep changes requested in a single PR as simple as possible

Reviewer: ```suggestion
- Keep the changes requested in a single PR as simple as possible
```

- To make it easier to trace back the history after a squash merge, please split a PR if it contains changes with different content.
  - For example, if you are adding multiple features, or adding features and refactoring at the same time, each should be a separate PR.

---

> # Contribution Guide

Reviewer: ```suggestion
# Contribution Guide (JA)
```
